src/screens/OrgList/OrgList.tsx

```diff
@@ -643,53 +643,40 @@ function orgList(): JSX.Element {
         </Modal>{' '}
         {/* Plugin Notification Modal after Org is Created */}
         <Modal show={dialogModalisOpen} onHide={toggleDialogModal}>
+          <Modal.Header
+            className="bg-primary"
+            closeButton
+            data-testid="modalOrganizationHeader"
```

Comment:
Rename the `data-testid`. It's already used for other modal above in the same component.